packages/desktop-client/src/components/table.tsx

```diff
@@ -366,7 +365,7 @@ type InputCellProps = ComponentProps<typeof Cell> & {
   inputProps: ComponentProps<typeof InputValue>;
   onUpdate: ComponentProps<typeof InputValue>['onUpdate'];
   onBlur: ComponentProps<typeof InputValue>['onBlur'];
-  textAlign?: string;
+  textAlign?: CSSProperties['textAlign'];
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏nice!